src/imgtools/cli/niftiindex.py

```
@@ -0,0 +1,142 @@
+        force=force,
+    )
+    try:
+        crawler.crawl()
+    except CrawlerOutputDirError as e:
+        logger.exception("Output directory error")
+        # exit with a non-zero status code
+        raise click.ClickException(f"Output directory error: {e}") from e
+    except Exception as e:
+        logger.exception("Unknown Crawling Error has occurred")
+        # exit with a non-zero status code
+        raise click.ClickException(f"Crawling failed") from e
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)